src/pmdas/bpf/modules/GNUmakefile

```diff
@@ -9,114 +9,28 @@ MODULEDIR = $(PCP_PMDASADM_DIR)/$(IAM)/modules
 MODULES	= \
 	biolatency.so \
 	runqlat.so    \
-	execsnoop.so  \
+
+# conditional enabled bpf PMDA modules
+## modules requiring libbpf >= 0.7.0
+ifeq ($(shell printf "%s\n" 0.7.0 `pkg-config --modversion libbpf` | sort --version-sort | head -n1),0.7.0)
```

Comment:
This sort of thing would usually be worked out by the configure script (see the Qt version checks, for an example) - its often handy to print out the version there too - and not by individual Makefiles during the build.

Reply:
Good point, I've moved the logic to the configure script now and added additional output in case the execsnoop module is not included because the libbpf version is too old.